src/services/users.services.ts

```diff
@@ -1,15 +1,52 @@
-import {User} from '../models/User';
-import {db} from '../database';
+import axios from 'axios';
+import config from '../config';
 
-const userRepository = db.getRepository(User);
+interface User {
+  id: string,
+  username: string,
+  name: string,
+  email: string
+  sex: string,
+  campus: string,
+  faculty: string,
+}
 
-const getUsers = async () => {
+const mainInstance = axios.create({
+  baseURL: config.mainApiUrl,
+});
+
+const getUser = async (token: string): Promise<User> => {
   try {
-    const users = await userRepository.find();
-    return users;
+    const response = await mainInstance.get('/users/my-account', {
```

Comment:
Ini endpoint `/users/my-account` udah di tes bisa di local lu chriss? Kemarin gua sempet gabisa, dan bisanya malah di endpoin `/users/me`. Kalau misal bisa ignore this comment ajaah dan lgsg di-resolve